### PR TITLE
Project structure upgrade

### PR DESCRIPTION
**Pull request by kotsabiukmv98**

### CODE REVIEW

File: frontend/src/app/modules/workspace/file-browser-section/file-browser-section.component.ts [hunk 1]
```diff
@@ -152,7 +157,8 @@ export class FileBrowserSectionComponent implements OnInit {
 
     nodeSelect(evt: any): void {
         const nodeSelected: TreeNode = evt.node;
-        if (nodeSelected.data !== 'Folder') {
+        if (nodeSelected.type === TreeNodeType.file.toString()) {
+            debugger;
```
Comment (Potapy4) [inline frontend/src/app/modules/workspace/file-browser-section/file-browser-section.component.ts]: no debugger, please

File: frontend/src/app/services/project-structure-formater.service.ts [hunk 2]
```diff
@@ -3,43 +3,50 @@ import { Injectable } from '@angular/core';
...
+    toTreeView(projectStructure: ProjectStructureDTO) : TreeNode {
+        let root : TreeNode;
+        root = this.toTreeNode(projectStructure.nestedFiles[0]);
+        return root;
+    }
+
+    toTreeNode(fileStructure: FileStructureDTO, parent: TreeNode = null) : TreeNode {
+        let root : TreeNode;
+        if (fileStructure.type == TreeNodeType.folder){
+            debugger;
```
Comment (Potapy4) [inline frontend/src/app/services/project-structure-formater.service.ts]: same here

File: frontend/src/app/services/project-structure-formater.service.ts [hunk 3]
```diff
@@ -51,7 +58,8 @@ export class NodesPrepareToViewService {
     makeFolderNode(name: string, id: string) {
         const emptyFolder = this.getEmptyFolderNode();
         emptyFolder.label = name;
-        emptyFolder.key = id;
+        debugger;
```
Comment (Potapy4) [inline frontend/src/app/services/project-structure-formater.service.ts]: same here

File: frontend/src/app/modules/workspace/file-browser-section/file-browser-section.component.ts (hunk 1 above)
Comment (kotsabiukmv98) [reply frontend/src/app/modules/workspace/file-browser-section/file-browser-section.component.ts]: there is a lot of time yet before morning, in my last commit I will use ctrl + f for finding all occurrences of "debugger")

**Pull request merged**